contribute.html

```diff
@@ -50,10 +50,11 @@
 					<a class="nav-link" href="/">HOME</a>
 					<a class="nav-link" href="/#features">FEATURES</a>
 					<a class="nav-link" href="/#download">DOWNLOAD</a>
+					<a class="nav-link" href="https://dietpi.com/docs/">DOCUMENTATION</a>
```

Comment:
```suggestion
					<a class="nav-link" href="https://dietpi.com/docs/">DOCS</a>
```
I guess even with this, the navigation can be too large in some cases, so I might need to adjust CSS. Will test this separately.